src/unstract/sdk/exceptions.py

```diff
@@ -39,3 +39,18 @@ class OCRError(SdkError):
 
 class RateLimitError(SdkError):
     DEFAULT_MESSAGE = "Running into rate limit errors, please try again later"
+
+
+class FileStorageError(SdkError):
+    DEFAULT_MESSAGE = (
+        "Error while connecting with the storage. "
+        "Please check the configuration credentials"
+    )
+
+
+class FileOperationError(SdkError):
+    DEFAULT_MESSAGE = (
+        "Error while performing operation on the file. "
+        "Please check specific storage error for "
+        "further information"
+    )
```

Comment:
@gaya3-zipstack is there any reason why we need to differentiate both these types of errors? How do you think it might help downstream usecases?
We could try to have a common subclass of `SdkError` that both these classes can inherit from to simplify the error handling maybe

Reply:
FileOperationError happens when we try to work on a file
FileStorageError happens when we try to get the storage connection.
Common exception - Do you  mean that this will help handling at the caller's side @chandrasekharan-zipstack ?

Reply:
Will take up any improvements incrementally